pydsa/count_inversions.py

```diff
@@ -0,0 +1,45 @@
+	# Counting inversions in an array
+	# Complexity : O(nlog(n))
+
+
+def _merge(L, R):
+    m, i, j, cnt = [], 0, 0, 0
+    while i < len(L) and j < len(R):
+        if L[i] <= R[j]:
+            m.append(L[i])
+            i += 1
+        else:
+            m.append(R[j])
+            j += 1
+            cnt = cnt + len(L) - i
+    if i == len(L):
+        m.extend(R[j:])
+    else:
+        m.extend(L[i:])
+    return m, cnt
+
+
+def merge_sort(a):
```

Comment:
@aktech The function there returns the sorted array and not the count of inversions.